src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/weapons/ExplosiveBow.java

```diff
@@ -56,21 +56,20 @@ public BowShootHandler onShoot() {
             for (Entity nearby : entites) {
                 LivingEntity entity = (LivingEntity) nearby;
 
-                Vector distanceVector = entity.getLocation().toVector().subtract(target.getLocation().toVector());
-                distanceVector.setY(distanceVector.getY() + 0.6);
-
-                Vector velocity = entity.getVelocity();
+                Vector distanceVector = entity.getLocation().toVector().subtract(target.getLocation().toVector())
+                    .add(new Vector(0, 0.75, 0));
```

Comment:
Format consistentlyy
```suggestion
                Vector distanceVector = entity.getLocation().toVector()
                    .subtract(target.getLocation().toVector())
                    .add(new Vector(0, 0.75, 0));
```